scripts/coverage/python_coverage.sh

```diff
@@ -79,7 +79,7 @@ check_report() {
 
 cleanup
 check_prerequisite
-run_ut
+# run_ut
```

Comment:
@pranavPawar07  if we are not running UT then how we are collecting coverage report ?
Could you pls check what is the coverage percentage in report in both cases e..g without running UT and with UT run.